packages/sdk/src/server/index.ts

```diff
@@ -491,47 +517,25 @@ export class ServerClient {
    * @template T - The expected response type.
    * @param path - The API endpoint path.
    * @param opts - The options for the request.
-   * @param authType - The type of authorization to use ("oauth" or "connect").
    * @returns A promise resolving to the API response.
    * @throws Will throw an error if the response status is not OK.
    */
-  private async makeAuthorizedRequest<T>(
+  public async makeAuthorizedRequest<T>(
```

Comment:
should be public since API clients should be able to easily make any authorized request against the API, especially since we don't support built-in methods for the rest of the REST API.